compose/compose-file/compose-versioning.md

```diff
@@ -411,6 +412,27 @@ It's more complicated if you're using particular configuration features:
           data:
             external: true
 
+## Compatibility mode
+
+`docker-compose` 1.20.0 introduces a new `--compatibility` flag designed to
+help developers transition to version 3 more easily. When enabled,
+`docker-compose` reads the `deploy` section of each service's definition and
+attempts to translate it into the equivalent version 2 parameter. Currently,
+the following deploy keys are translated:
+
+- [resources](index.md#resources) limits and memory reservations
+- [replicas](index.md#replicas)
+- [restart_policy](index.md#restartpolicy) `condition` and `max_attempts`
+
+All other keys are ignored and produce a warning if present. You can review
+the configuration that will be used to deploy by using the `--compatibility`
+flag with the `config` command.
+
+> **Note:** We recommend against using `--compatibility` mode in production.
```

Comment:
@shin- can you use the new note format. I should look like this

```
> Don't use this in production
> You shouldn't use `--compatibility` mode in production
> because....
```